mopidy/core/mixer.py

```diff
@@ -40,13 +44,22 @@ def get_mute(self):
         :class:`True` if muted, :class:`False` unmuted, :class:`None` if
         unknown.
         """
-        if self._mixer is not None:
+        if self._mixer is None:
+            return self._mute
+        else:
             return self._mixer.get_mute().get()
 
     def set_mute(self, mute):
         """Set mute state.
 
         :class:`True` to mute, :class:`False` to unmute.
+
+        Returns :class:`True` if call is successful, otherwise :class:`False`.
         """
-        if self._mixer is not None:
-            self._mixer.set_mute(bool(mute))
+        if self._mixer is None:
+            self._mute = mute
```

Comment:
Because you can mute the output - and the state is needed when ``get_mute`` is called. The ``mute_changed`` event could get the value directly from the parameters, though.